students/byoungju/users/views.py

```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+
+
+class UserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        if ((data["email"] or data["password"]) not in data.values()):
+            return JsonResponse({"message":"KEY_ERROR"}, status = 400)
+
+        if (re.match("\w*@\w*.\w*", data["email"]) == None):
+            return JsonResponse({"message":"INVALID__EMAIL"}, status = 400)
+
+        if (re.match("[\w\W{8,20}]", data["password"]) == None):
+            return JsonResponse({"message":"INVALID_PASSWORD"}, status = 400)  
```

Comment:
None 비교는 주로 is를 사용합니다!
```suggestion
        if (re.match("\w*@\w*.\w*", data["email"]) is None):
            return JsonResponse({"message":"INVALID__EMAIL"}, status = 400)

        if (re.match("[\w\W{8,20}]", data["password"]) is None):
            return JsonResponse({"message":"INVALID_PASSWORD"}, status = 400)  
```